### PR TITLE
chore: add MIW wallet creation

### PR DESCRIPTION
**Pull request by ds-jhartmann**
## Description
- add seeding job for MIW wallet creation. 
- Full EDC flow between dataconsumer and dataprovider was tested successfully (Catalog Request -> Contract Negotiation -> Transfer Process -> Dataplane Request)

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by evegufy — CHANGES_REQUESTED** [review]
Looks very good in general, just a linter issue and could you please also configure the newly added dataconsumerTwo? thank you!

File: charts/umbrella/values.yaml [hunk 1]
```diff
@@ -772,20 +795,20 @@ dataconsumerTwo:
...
       ssi:
         miw:
           url: http://managed-identity-wallets.tx.test
-          authorityId: BPNL00000003CRHK
+          authorityId: *authority-bpn
         oauth:
           tokenurl: http://centralidp.tx.test/auth/realms/CX-Central/protocol/openid-connect/token
           client:
-            id: *miw_client
+            id: satest08
```
Comment (evegufy) [inline charts/umbrella/values.yaml:811]: could you please take satest03, same as in adopter-dataexchange values file

File: charts/umbrella/values.yaml [hunk 2]
```diff
@@ -772,20 +795,20 @@ dataconsumerTwo:
   seedTestdata: false
   nameOverride: dataconsumer-2
   secrets:
-    edc-miw-keycloak-secret: changeme  # TODO switch to existing user
+    edc-miw-keycloak-secret: WUXpQx1aIclA7enqtk4o2uvLDLMreUMI
```
Comment (evegufy) [inline charts/umbrella/values.yaml:798]: could you please take tPwy4exxH1sXBRQouobSA2nNVaaPuwCs , same as in adopter-dataexchange values file

File: charts/umbrella/values.yaml [hunk 3]
```diff
@@ -772,20 +795,20 @@ dataconsumerTwo:
...
   tractusx-connector:
     nameOverride: dataconsumer-2-edc
     participant:
-      id: BPNL000000000000
+      id: BPNL00000003B6LU
```
Comment (evegufy) [inline charts/umbrella/values.yaml:802]: could you please take BPNL00000003AVTH, same as in adopter-dataexchange values file

**Review by evegufy — CHANGES_REQUESTED** [review]

File: charts/umbrella/values.yaml [hunk 4]
```diff
@@ -539,8 +539,31 @@ sdfactory:
     # -- Details for Clearing House Client Secret
     clearingHouseClientSecret: ""
 
+# -- Map of company BPN and company name. For each of these Companies, a MIW wallet will be created.
+companies:
```
Comment (evegufy) [inline charts/umbrella/values.yaml:543]: Might make sense to move this map to Values.managed-identity-wallet.seeding, as it belongs to the miw seeding

**Review by evegufy — CHANGES_REQUESTED** [review]
I tested it and it works great, just one comment to tidy up a bit

File: charts/umbrella/values.yaml (hunk 4 above)
Comment (ds-jhartmann) [reply charts/umbrella/values.yaml]: Good point. Adjusted it

**Review by evegufy — APPROVED** [review]

**Pull request merged**